server/glsp-api/src/main/java/com/eclipsesource/glsp/api/action/kind/ServerStatusAction.java

```diff
@@ -21,6 +21,7 @@
 public class ServerStatusAction extends Action {
 	private String severity;
 	private String message;
+	private String details;
 
 	public ServerStatusAction() {
```

Comment:
Please restore the `hashCode` and `equals` method of this class with the new field ;-)